src/components/PageHeader/NewClassroomModal.tsx

```diff
@@ -0,0 +1,131 @@
+import { LoadingButton } from '@mui/lab'
+import { Box, TextField, Button } from '@mui/material'
```

Comment:
o primeiro não tenho certeza, mas verifica se tem como achar ele depois do `/lab`
```suggestion
import LoadingButton from '@mui/lab/LoadingButton'
import Box from '@mui/material/Box'
import TextField from '@mui/material/TextField'
import Button from '@mui/material/Button'
```